controllers/apply-next/form-router.js

```diff
@@ -217,20 +223,84 @@ function initFormRouter(formModel) {
         });
     });
 
+    function renderError(error, req, res) {
+        const errorCopy = req.i18n.__('apply.error');
+        res.render(path.resolve(__dirname, './views/error'), {
+            error: error,
+            title: errorCopy.title,
+            errorCopy: errorCopy,
+            returnUrl: `${req.baseUrl}/review`
+        });
+    }
+
+    function flattenFormData(formData) {
```

Comment:
In the original code this was a one-liner:

           function flattenFormData(formData) {
                return Object.assign({}, ...flatMap(formData));
            }

... but this doesn't work with the new structure. Maybe there's a better method?!